x-pack/plugins/ml/server/models/data_recognizer/modules/security_linux/ml/v3_linux_anomalous_network_activity.json

```diff
@@ -0,0 +1,62 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Security: Linux - Looks for unusual processes using the network which could indicate command-and-control, lateral movement, persistence, or data exfiltration activity.",
+  "groups": [
+    "auditbeat",
+    "endpoint",
+    "linux",
+    "network",
+    "security"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "Detects rare process.name values.",
+        "function": "rare",
+        "by_field_name": "process.name"
+      }
+    ],
+    "influencers": [
+      "host.name",
+      "process.name",
+      "user.name",
+      "destination.ip"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "64mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
```

Comment:
The `custom_settings` for all the `security_linux` jobs is malformed - they contain an extra` custom_settings` level inside `custom_settings`, and the JSON is malformed, which prevents the jobs loading correctly in the ML job wizard. I see this error in the Kibana server console:

```
[2022-05-17T15:19:05.344+01:00][WARN ][plugins.ml] Data recognizer error loading config for job v3_linux_anomalous_network_port_activity for module security_linux_v3. SyntaxError: Unexpected end of JSON input
[2022-05-17T15:19:05.344+01:00][WARN ][plugins.ml] Data recognizer error loading config for job v3_linux_network_configuration_discovery for module security_linux_v3. SyntaxError: Unexpected end of JSON input
[2022-05-17T15:19:05.345+01:00][WARN ][plugins.ml] Data recognizer error loading config for job v3_linux_network_connection_discovery for module security_linux_v3. SyntaxError: Unexpected end of JSON input
```

Reply:
done